data/json/recipes/recipe_deconstruction.json

```diff
@@ -626,8 +626,8 @@
     "difficulty": 7,
     "skills_required": [ "firstaid", 5 ],
     "time": 50000,
+    "using": [ [ "soldering_standard", 20 ] ],
```

Comment:
Remove this and all the other changes unrelated to the cardboard thing.

Reply:
It's there from a Pull I did from the repo. I'll fix it tho.